addons/mail/static/src/components/follower/follower.scss

```diff
@@ -2,55 +2,22 @@
 // Layout
 // ------------------------------------------------------------------
 
-.o_Follower {
-    display: flex;
-    flex-flow: row;
-    justify-content: space-between;
-    padding: map-get($spacers, 0);
-}
-
 .o_Follower_avatar {
-    width: 24px;
-    height: 24px;
-    margin-inline-end: map-get($spacers, 2);
+    width: $o-mail-partner-avatar-size;
+    height: $o-mail-partner-avatar-size;
 }
 
 .o_Follower_details {
-    align-items: center;
-    display: flex;
-    flex: 1;
+    flex-basis: 0;
     min-width: 0;
-    padding-left: map-get($spacers, 3);
-    padding-right: map-get($spacers, 3);
 }
 
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
-.o_Follower_avatar {
-    border-radius: 50%;
-}
-
-.o_Follower_button {
-    border-radius: 0;
-
-    &:hover {
-        background: gray('400');
-        color: $black;
-    }
-}
-
-.o_Follower_details {
-    color: gray('700');
-
-    &:hover {
-        background: gray('400');
-        color: $black;
-    }
-
-    &.o-inactive {
-        opacity: 0.25;
-        font-style: italic;
-    }
+.o_Follower_button:hover,
+.o_Follower_details:hover {
+    background: $gray-400;
+    color: $black;
```

Comment:
Isn't to apply a `bg-white` class enough to achieve the hover state?

Reply:
The hover state of `bg-white` is too light and not contrasty enough compared to `$gray-400`.
Perhaps we can improve the design once the SCSS optimization is done ?

![demo_followers](https://user-images.githubusercontent.com/80679690/162228572-6c989d85-3920-435f-a9d4-44815168c7a1.jpg)
.

Reply:
😕 mmm... 
At least the first one is readable, but as said multiple times improvements are out-of-scope for these tasks.
Ok to review after SCSS optimisation 👍 

added to task-2818586